packages/expo-eas-client-id/ios/EASClientId/EASClientIdModule.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+public class EASClientIdModule: Module {
+  public func definition() -> ModuleDefinition {
+    name("EASClientId")
+
+    function("getClientIdAsync") {
+      EASClientId.uuid.uuidString
+    }
```

Comment:
Maybe it makes more sense to move this to constants?

```suggestion
    constants([
      "clientId": EASClientId.uuid.uuidString
    ])
```